docs/getting-started/running/index.md

```diff
@@ -0,0 +1,39 @@
+---
+title: Running the Application
+parent: Getting Started
+---
+
+## Basic Development
+
+To run the application, you will need to complete the [Setup steps](../setup). Once you have completed the Setup steps, you can run the application as described below.
+
+First start the database service using Podman Compose. From the root of the project, run the following command:
+
+```shell
+podman-compose up -d
+```
+
+Then run the application:
+
+```shell
+./gradlew :server:run
```

Comment:
Is there no need for the `run.sh` script?

Reply:
The operative part of `run.sh` is the `:server:run` gradle task. I've updated the notes just below this command to make that more clear, so it's less of a mystery of what the script is doing.